appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/YaFormEditor.java

```diff
@@ -210,6 +207,22 @@ public boolean shouldDisplayHiddenComponents() {
 
   // FileEditor methods
 
+  @Override
+  public DropTargetProvider getDropTargetProvider() {
```

Comment:
See my note on YaBlocksEditor about the fact that this method is not used anywhere.

Reply:
In particular, it seems the superclass defines the abstract method, but no where in the code base is it called.